iTaylorCodes/Capstone-2-Neighborhood-fe
# Add login/logout Feature

### What does this PR do?
- This PR sets up the user login and logout features
- This PR closes #15 
### Description of Task to be completed
- Setup `LoginForm` component. Setup `userContext`.
### How should this be manually tested?
- Visit /login, fill out login form, verify the navbar has updated to show a logged in user. Then visit /logout to logout.

https://user-images.githubusercontent.com/76837421/145337522-40149dfe-18c5-4cba-be0b-46bea2433fc3.mov
